src/clj/rems/poller/email.clj

```diff
@@ -32,7 +38,8 @@
       :subject (text :t.email.application-approved/subject)
       :body (text-format :t.email.application-approved/message
                          (:userid member)
-                         (:id application))})))
+                         (:application/id event)
+                         (link-to-application (:id application)))})))
```

Comment:
`(:application/id event)` and `(:id application)` are interchangeable. Sticking to just one of them everywhere would improve consistency. Maybe prefer `(:application/id event)` to reduce dependency on old application schema?

Reply:
whoops, my bad